try.ps1

```diff
@@ -0,0 +1 @@
+cd c:\p\profiler; get-module profiler | remove-module ; import-module .\Profiler\Profiler.psd1; $l = Trace-Script { function a {  1..10 | % { $ps = get-service; if ($_ -eq 4) { [gc]::Collect($true) } } } ; a } ; $l.Events | select index, @{ n="txt"; e = { ($_.text[0..20] -join "" ) }}, heapsize, workingset, selfheapsize, selfworkingset, AllocatedBytes, SelfAllocatedBytes, gc0, gc1, gc2, selfgc0, selfgc1, selfgc2 | ft
```

Comment:
Try it like this, there is no summary yet, but we can see that the heap size is growing, then it becomes smaller (the selfHeapSize is negative), and we can also see that gc count is increased,   because GC happened and compacted the heap.